python-sdk/docs/configurations.rst

```diff
@@ -27,11 +27,96 @@ We can also configure the default schema specific to the database type (example:
 
 .. code:: python
 
-   AIRFLOW__ASTRO_SDK__POSTGRES_DEFAULT_SCHEMA = "tmp"
+   AIRFLOW__ASTRO_SDK__POSTGRES_DEFAULT_SCHEMA = "postgres_tmp"
+   AIRFLOW__ASTRO_SDK__BIGQUERY_DEFAULT_SCHEMA = "bigquery_tmp"
+   AIRFLOW__ASTRO_SDK__SNOWFLAKE_DEFAULT_SCHEMA = "snowflake_tmp"
+   AIRFLOW__ASTRO_SDK__REDSHIFT_DEFAULT_SCHEMA = "redshift_tmp"
 
 or by updating Airflow's configuration
 
 .. code:: shell
 
    [astro_sdk]
-   postgres_default_schema = "tmp"
+   postgres_default_schema = "postgres_tmp"
+   bigquery_default_schema = "bigquery_tmp"
+   snowflake_default_schema = "snowflake_tmp"
+   redshift_default_schema = "redshift_tmp"
+
+
+Configuring the Custom XCom Backends
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can also examine Airflow’s configuration:
+
+.. code:: shell
+
+   AIRFLOW__CORE__XCOM_BACKEND="tmp"
+
+or by updating Airflow's configuration
+
+.. code:: shell
+
+   [astro_sdk]
+   xcom_backend = "tmp"
+
+
+Configuring the unsafe dataframe storage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code:: shell
```

Comment:
We should add a description on when a user should change this setting/config

Reply:
@sunank200 this setting is more of content from user because the dataframes are pickled and saved in the airflow database, which is a security risk because of pickling and also from a size perspective since we don't control the size of dataframe, if the dataframe is big enough, it can potentially break the Airflow's metadata.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/861/commits/68f2bf333711b3c05666e56fb61073fe3853cb84)